src/silo/common/string_utils.cpp

```diff
@@ -19,4 +19,12 @@ std::vector<std::string> splitBy(const std::string& value, const std::string_vie
    return splits;
 }
 
+std::string removeSymbol(const std::string& value, const std::string_view symbol) {
+   std::string result = value;
+   const auto symbol_position = result.find(symbol);
+   if (symbol_position != std::string::npos) {
+      result.erase(symbol_position, symbol.size());
+   }
+   return result;
```

Comment:
This only removes up to one quote. Lets also cover the case where on string is completely quoted

```
return value.erase(
      std::remove(value.begin(), value.end(), '\"'),
      value.end()
   );
```

Reply:
(remove all " from the string, not only the first one found)

Reply:
I changed it, so it removes all. Additionally, I added tests to verify the behavior.